src/main/java/io/chazza/advancementapi/AdvancementAPI.java

```diff
@@ -238,14 +252,13 @@ public String toString() {
         }
     }
 
-
     //BEGIN CLASSES
     @Builder(builderMethodName = "hiddenbuilder")
     public static class Condition {
         protected String name;
-        protected JSONObject set;
+        protected JsonObject set;
```

Comment:
This should be an object to allow the ability to add a string so trigger types such as Minecraft:brewed_potion can have 'potion' as a condition. I may be wrong.